modules/dnn/src/layers/fast_convolution/depthwise_convolution.cpp

```diff
@@ -295,14 +319,16 @@ static void depthWiseBlockConv1D(const float* wptr,
             s2 = 0.f;
         }
         out = imgptr0[in_j0]*w00*s0 + imgptr0[in_j1]*w01*s1 + imgptr0[in_j2]*w02*s2 + bias;
+        if (fusedAdd)
+            out += outptr[out_j];
         if (relu)
             out = out > 0.f ? out : out*relu_coeff;
-        outptr[out_j] = out;
+        outptr[out_j] += out;
```

Comment:
Seems suspicious - we add outptr 2 times, is that correct?

Reply:
Thx for reviewing. Fixed.